### PR TITLE
Fixing the whitespace&newlines in xml

### PR DESCRIPTION
**Pull request by hub-cap**

### CODE REVIEW

**Comment by hub-cap** [comment]
see #124 for more info

File: reddwarf/common/wsgi.py [hunk 1]
```diff
@@ -261,6 +261,13 @@ def __init__(self, metadata=None):
         metadata['plurals'] = CUSTOM_PLURALS_METADATA
         super(ReddwarfXMLDeserializer, self).__init__(metadata)
 
+    def default(self, datastring):
+        # Sanitize the newlines
+        # hub-cap: This feels wrong but minidom keeps the newlines
```
Comment (ed-) [inline reddwarf/common/wsgi.py:266]: `This feels wrong`

_DOES IT?!?_

**Comment by rnirmal** [comment]
+1

**Comment by cp16net** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Comment by ed-** [comment]
blargh but +1

**Pull request merged**